Resources/doc/installation.md

```diff
@@ -69,6 +69,35 @@ sulu_article:
     prefix: /admin/articles
 ```
 
+### Webspace behavior
+
+When you have more than one webspace in your Sulu project you must provide a default main webspace and optional
+you can also provide default additional webspaces.
+
+#### Functionality
+
+An article is delivered only on the main webspace.
+Additionally the article can be delivered on additional webspaces marked as copy of the main one to avoid duplicate content issues.
+This is done with the `canonical tag` in the html header.
+
+More information about this:
+* Duplicate content: https://moz.com/learn/seo/duplicate-content
+* Canonicalization: https://moz.com/learn/seo/canonicalization
+
+#### Configuration
+```yml
+sulu_article:
+    default_main_webspace: 'webspace1'
+    default_additional_webspaces:
+        - 'webspace2'
+        - 'webspace3'
+```
+
+#### Content Manager
+
+This default webspace settings can be overwritten by the content manager for each article.
```

Comment:
For each article the default webspace configuration can be overwritten by the content manager.